.github/wiki/Home.md

```diff
@@ -1 +1,4 @@
-This repository contains infrastructure configuration for the [demo application](https://github.com/Nihisil/nimble-devops-ic-app).
\ No newline at end of file
+This repository contains infrastructure configuration for the [demo application](https://github.com/Nihisil/nimble-devops-ic-app).
```

Comment:
There should be a `_sidebar.md` file so you can organize the content that follow our convention
https://nimblehq.co/compass/development/documentation/github-wiki/#organization
example:
- https://github.com/nimblehq/worpt-api/blob/main/.github/wiki/_Sidebar.md
- https://github.com/nimblehq/worpt-api/wiki

Reply:
Thank you. It was added here: https://github.com/Nihisil/nimble-devops-ic-infrastructure/pull/17/commits/6961c3f66658ed1f631500886fee585934faa1ec